R/utils.R

```diff
@@ -15,6 +15,26 @@ check_pkg_installed <- function(pkg, call = rlang::caller_env()) {
   }
 }
 
+check_gdal_version <- function(min_version = "3.1") {
+    terra_gdal <- numeric_version(terra::gdal(lib = "gdal"))
+    version <- numeric_version(min_version)
+    terra_gdal < min_version
```

Comment:
Also feels like the output of this is reversed.  It makes more sense to me if it would return FALSE when the min version is not met.  I.e. `terra_gdal >= min_version`